cfme/tests/configure/test_rest_access_control.py

```diff
@@ -819,3 +819,99 @@ def test_delete_users_from_collection(self, users):
             initialEstimate: 1/4h
         """
         delete_resources_from_collection(users)
+
+
+COMMON_FEATURES = [
+    "Services",
+    "Compute",
+    "Configuration",
+    "Networks",
+    "Storage",
+    "Control",
+    "Automation",
+    "Monitor",
+    "Help",
+    "API",
+    "Service UI",
+]
+
+FEATURES_511 = COMMON_FEATURES + [
+    "Overview",
+    "Migration",
+    "User Settings",
+    "All VM and Instance Access Rules",
+    "Main Configuration",
+    "Red Hat Cloud",
+]
+
+FEATURES_510 = COMMON_FEATURES + [
+    "Cloud Intel",
+    "Red Hat Insights",
+    "Optimize",
+    "Access Rules for all Virtual Machines",
+]
+
+
+@pytest.fixture(params=[True, False], ids=["individual", "everything"])
+def create_role(appliance, request):
+    role_name = "API-role-{}".format(fauxfactory.gen_alpha(4))
```

Comment:
optional: start using `fauxfactory` kwargs like `start` 
```
fauxfactory.gen_alpha(15, start="API-role-")
```

Reply:
That's new!